### PR TITLE
feat(sbb-header-environment): add 'inte' as a recognized environment

### PR DESCRIPTION
**Pull request by fritteli**
Some INT environments are called "inte" instead of "int"; add a corresponding style class.

### CODE REVIEW

**Review by kyubisation — CHANGES_REQUESTED** [review]
Please follow our guideline for commit messages and PR title: https://github.com/sbb-design-systems/sbb-angular/blob/main/CONTRIBUTING.md#commit
Other than that, looks good to me.

File: src/angular/header-lean/header.scss [hunk 1]
```diff
@@ -61,7 +61,7 @@
 .sbb-header-environment-edu {
   background-color: var(--sbb-color-green);
 }
-.sbb-header-environment-int {
+.sbb-header-environment-int, .sbb-header-environment-inte {
```
Comment (kyubisation) [inline src/angular/header-lean/header.scss:64]:
```suggestion
.sbb-header-environment-int,
.sbb-header-environment-inte {
```

**Comment by fritteli** [comment]
@kyubisation thanks for fixing the code style error yourself :)

**Review by kyubisation — APPROVED** [review]
I will merge it on monday.
Thank you for the contribution. 😃

**Pull request merged**